test/cli.js

```diff
@@ -6,21 +6,26 @@ import execa from 'execa';
 
 process.chdir(__dirname);
 
+const cli = (...args) => {
+	const cliPath = path.join(__dirname, '../cli.js');
```

Comment:
Nah. It's not used outside the function, so better to keep it in context and it's doesn't really affect performance to keep it there.